lib/automation/report_runner.rb

```diff
@@ -70,6 +70,14 @@ def initialize(
 
       def run!
         start_date = (@offset + @days).days.ago
+
+        @title =
+          @title.gsub(
+            "%DATE%",
+            start_date.strftime("%Y-%m-%d") + " - " +
+              (start_date + @days.days).strftime("%Y-%m-%d"),
```

Comment:
I was going to suggest `start_date.iso8601`, but realized `start_date` is actually a `DateTime`. Maybe we could convert `to_date` or rename the variable.